improver_tests/nbhood/nbhood/test_GeneratePercentilesFromANeighbourhood.py

```diff
@@ -387,18 +386,38 @@ def test_single_point(self):
         ).process(cube)
         self.assertArrayAlmostEqual(result.data, expected)
 
-    def test_multi_point_multitimes(self):
-        """Test behaviour for points over multiple times."""
+    def test_single_point_single_percentile(self):
```

Comment:
Added a new simple test for a single percentile. This was just to make sure it worked for this case and it seems fine so left it in.

Reply:
Now this test returns a cube with a scalar percentile dimension. Also added a test for a single realization dimension